lib/internal/Magento/Framework/View/Asset/ConfigInterface.php

```diff
@@ -15,28 +18,40 @@
     /**
      * Check whether merging of CSS files is on
      *
+     * @param string $scopeType
+     * @param string|null $scopeCode
+     *
      * @return bool
      */
-    public function isMergeCssFiles();
+    public function isMergeCssFiles($scopeType = ScopeInterface::SCOPE_STORE, $scopeCode = null);
```

Comment:
This is forbidden change because of [backward compatibility policy](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/#adding-parameters-in-public-methods).

You may:
- introduce new interface with *required arguments* more
- introduce new class that will implement created interface
- inject  new class to `Magento\Framework\View\Asset\Config` and delegate all calls to new class with suitable arguments
- declare Magento\Framework\View\Asset\Config as `@deprecated`
- inject new interface in addition to `Magento\Framework\View\Asset\ConfigInterface` where is needed (backward compatibility for modified constructor should be preserved)
- do not remove `Magento\Framework\View\Asset\ConfigInterface` from any constructors, mark that argument/property as `@deprecated` instead